polaris.shopify.com/content/whats-new/version-11-color.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Version 11 Color
+description: Learn about what changes were introduced to color in Polaris v11.
+imageUrl: /images/updates/new-colors-cover.png
+keywords:
+  - color
+  - tokens
+---
+
+![Preview of new alias tokens](/images/updates/new-colors-cover.png)
+
+## Why are we making changes?
+
+Over the past year we have invested a lot of time and energy into creating a more robust and cohesive token system in Polaris. We have updated typography, spacing, shadows and breakpoints. All of this work has made it easier than ever to build with Polaris and has unlocked new levels of composability within the system itself. We knew we needed to bring this same thinking to our color tokens as well.
+
+These were the main drivers for this update.
+
+1. Create a defined color palette to generate tokens from. This will eliminate color sprawl and ensure a more consistent and cohesive use of color across the admin.
```

Comment:
```suggestion
1. Create a defined color palette with multiple lightness values that is used to create new tokens. This will eliminate color sprawl and ensure a more consistent and cohesive use of color across the admin.
```

We don't generate any tokens in our system. They are all chosen with strong design opinions. Correct me if I am wrong.